1/index.js

```diff
@@ -0,0 +1,400 @@
+            that.options.auto = false; that.options.arrows = false; that.options.dots = false;
+            that.leftArrow.style.display = 'none'; that.rightArrow.style.display = 'none'
+        };
+        if(that.elemCount >= 1) {   // show first element
+
+            that.sldrElemFirst.style.display = 'block';
+        };
+
+        if(!that.options.loop) {
+            that.leftArrow.style.display = 'none';  // disable left arrow
+            that.leftArrow.style.opacity = '0.5';  // disable left arrow
+            that.options.auto = false; // disable auto autoscroll
+        }
+        else if(that.options.auto) {   // autoscroll initialization
+            setAutoScroll();
+            // Stop scrolling when hovering over an element
+            that.sldrList.addEventListener('mouseenter', function() {clearInterval(that.autoScroll)}, false);
+            that.sldrList.addEventListener('mouseleave', setAutoScroll, false)
+        };
+
+        if(that.options.arrows) {  // arrows initialization
+            that.leftArrow.addEventListener('click', function() {
+                let fnTime = getTime();
+                if(fnTime - bgTime > 1000) {
+                    bgTime = fnTime; that.elemPrev()
+                }
+            }, false);
+            that.rightArrow.addEventListener('click', function() {
+                let fnTime = getTime();
+                if(fnTime - bgTime > 1000) {
+                    bgTime = fnTime; that.elemNext()
+                }
+            }, false)
+        }
+        else {
+            that.leftArrow.style.display = 'none'; that.rightArrow.style.display = 'none'
+        };
+
+        if(that.options.dots) {  // initialization dots indicators
+            let sum = '', diffNum;
+            for(let i=0; i<that.elemCount; i++) {
+                sum += '<span class="sim-dot"></span>'
+            };
+            that.indicatorDots.innerHTML = sum;
+            that.indicatorDotsAll = that.sldrRoot.querySelectorAll('span.sim-dot');
+            // Assigning the 'click' eventListener to the dots
+            for(let n=0; n<that.elemCount; n++) {
+                that.indicatorDotsAll[n].addEventListener('click', function() {
+                    diffNum = Math.abs(n - that.currentElement);
+                    if(n < that.currentElement) {
+                        bgTime = getTime(); that.elemPrev(diffNum)
+                    }
+                    else if(n > that.currentElement) {
+                        bgTime = getTime(); that.elemNext(diffNum)
+                    }
+                    // if n == that.currentElement nothing happens
+                }, false)
+            };
+            that.dotOff(0);  // dot[0] is on, rest - off
+            for(let i=1; i<that.elemCount; i++) {
+                that.dotOn(i)
+            }
+        }
+    };
+
+
+
+    //----------------------------------- END SLIDER----------------------------------
```

Comment:
please prettify your code, you should have consistent indents between functions and vars, you do not need three lines between them, enough just one and blocks separators are so long

Reply:
done